moochub-schema.json

```diff
@@ -1,555 +1,3105 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
-  "$id": "https://github.com/moochub/schema/releases/tag/v2.3",
+  "$id": "https://github.com/moochub/schema/releases/tag/v3.0",
   "title": "JSON schema for MOOC providers integrated into moochub.org",
   "description": "This schema specifies the JSON format each MOOC provider must offer in order to be integrated into moochub.org",
   "type": "object",
   "properties": {
-    "links": {
-      "title": "Links",
+    "id": {
+      "type": "string",
+      "description": "The ID of the course as an UUID.",
+      "minLength": 1,
+      "example": "8978f51e-770c-4d0a-a2e3-23fae8f6aa79"
+    },
+    "type": {
+      "type": "string",
+      "description": "Type of the object.",
+      "enum": [
+        "Course"
+      ]
+    },
+    "attributes": {
       "type": "object",
-      "description": "Page information for the API",
+      "description": "The Attributes of the object.",
       "required": [
-        "self",
-        "first",
-        "last"
+        "name",
+        "learningResourceType",
+        "publisher",
+        "url",
+        "license",
+        "creator"
       ],
       "properties": {
-        "self": {
+        "name": {
           "type": "string",
-          "format": "iri",
-          "description": "The current page"
+          "description": "The title of the course.",
+          "example": "Confidential Communication in the Internet"
         },
-        "first": {
-          "type": "string",
-          "format": "iri",
-          "description": "The first page"
+        "courseCode": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "An internal short code of the course. Also possible to give internal codes for templates here.",
+          "example": "confidentialcommunication2021"
         },
-        "last": {
-          "type": "string",
-          "format": "iri",
-          "description": "The last page"
+        "learningResourceType": {
+          "type": "object",
+          "description": "The learning resource type as listed at https://w3id.org/kim/hcrt/scheme. This describes the resource according to HCRT (Higher Education Resource Type).",
+          "required": [
+            "identifier",
+            "type",
+            "inScheme"
+          ],
+          "properties": {
+            "identifier": {
+              "type": "string",
+              "description": "This is an IRI pointing at a node describing the learning resource type. Currently, only https://w3id.org/kim/hcrt/course is allowed.",
+              "format": "iri",
+              "enum": [
+                "https://w3id.org/kim/hcrt/course"
+              ]
+            },
+            "type": {
+              "type": "string",
+              "enum": [
+                "Concept"
+              ],
+              "description": "The type of the object according to AMB standard."
+            },
+            "inScheme": {
```

Comment:
```suggestion
            "scheme": {
```

Why the "in"?

Reply:
"inScheme" is the attribute name as specified by the AMB standard (OERSI) to which we want to comply.